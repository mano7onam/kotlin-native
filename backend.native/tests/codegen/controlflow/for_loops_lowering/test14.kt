package codegen.controlflow.for_loops_lowering.test14

import kotlin.test.*

@Test fun runTest() {
    val arr = arrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
    var sum = 0L
    for (i in 0L .. Long.MAX_VALUE step 10000000000000L) {
      for (j in arr.indices) {
        for (k in 'a' until 'z') {
          sum += i + j + k.toLong()
        }
      }  
    }
    println(sum)
}
