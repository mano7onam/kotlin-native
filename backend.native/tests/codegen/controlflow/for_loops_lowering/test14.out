4974102958245623758
